course/staff_views.py

```diff
@@ -164,17 +164,25 @@ def get_form_kwargs(self):
 
     def form_valid(self, form):
         if (self.instance.sis_id and form.cleaned_data["sis"]):
-            count = self.instance.enroll_from_sis()
-            if count >= 0:
+            addcount, delcount = self.instance.enroll_from_sis()
+            if addcount >= 0:
+                # ngettext works only with one parameter at a time, hence the complicated setup.
                 messages.info(
                     self.request,
                     format_lazy(
                         ngettext(
-                            'STUDENT_ENROLLED_FROM_SIS -- {count}',
-                            'STUDENTS_ENROLLED_FROM_SIS -- {count}',
-                            count
+                            'STUDENT_ENROLLED_FROM_SIS -- {addcount}',
+                            'STUDENTS_ENROLLED_FROM_SIS -- {addcount}',
+                            addcount
                         ),
-                        count=count,
+                        addcount=addcount,
+                    ) + " " + format_lazy(
```

Comment:
Concatenating translated strings with a space could maybe cause trouble with non-western languages that use right-to-left or top-to-bottom script, but let's ignore those now.

An alternative could be to render the two sentences in different paragraphs in the HTML.